playbooks/operationalizing/cloudforms.adoc

```diff
@@ -128,7 +128,7 @@ For installations that contain more than one appliance, repeat the above steps m
 
 Once the machine reboots, the web console should be accessible at the host public ip.
 
-https://<public_host>
+\https://<public_host>
```

Comment:
asciidoc automatically creates a hyperlink for it and htmlproofer blows up as it is not a valid address